pkg/resources/resources.go

```diff
@@ -865,17 +865,17 @@ const (
 )
 
 const (
-	// CSIMigrationWebhookName is the name of the csi-migration webhook service.
-	CSIMigrationWebhookName = "csi-migration-webhook"
-	// CSIMigrationWebhookSecretName defines the name of the secret containing the certificates for the csi-migration admission webhook.
-	CSIMigrationWebhookSecretName = "csi-migration-webhook-certs"
-
-	// CSIMigrationWebhookConfig is the name for the key that contains the webhook config.
-	CSIMigrationWebhookConfig = "webhook.config"
-	// CSIMigrationWebhookPort is the port used by the CSI-migration webhook.
-	CSIMigrationWebhookPort = 8443
-	// VsphereCSIMigrationWebhookConfigurationWebhookName is the webhook's name in the vSphere CSI_migration WebhookConfiguration.
-	VsphereCSIMigrationWebhookConfigurationWebhookName = "validation.csi.vsphere.vmware.com"
+	// VSphereCSINamespace is the namespace desired by vmware csi driver.
+	VSphereCSINamespace = "vmware-system-csi"
+	// VSphereCSIValidatingWebhookSVCName is the name of the validation webhook service.
+	VSphereCSIValidatingWebhookSVCName = "vsphere-webhook-svc"
```

Comment:
The name is already long enough, I don't think we need to use `SVC`. :wink: 

```suggestion
	VSphereCSIValidatingWebhookServiceName = "vsphere-webhook-svc"
```